src/Microsoft.Scripting/JavaScriptConverter.cpp

```diff
@@ -100,6 +100,10 @@ String^ JavaScriptConverter::ToString(IJavaScriptValue^ value)
         ObjCheckForFailure(JsStringToPointer(strRef, &str, &strLen));
         result = ref new String(str, strLen);
     }
+    else if (value->Type == JavaScriptValueType::Symbol)
```

Comment:
Should comment about why this is being done here -- that there isn't a way from the host to get the symbol description.